test/local/db_tests.js

```diff
@@ -217,7 +218,8 @@ test(
       name: '',
       type: 'mobile',
       pushCallback: 'https://foo/bar',
-      pushPublicKey: crypto.randomBytes(32)
+      pushPublicKey: base64url(crypto.randomBytes(65)),
```

Comment:
The p256dh (pushPublicKey) is in raw asn1 format, so the "\x04" is the semi-magical "This is a key pair" flag. 

Fun fact: some libraries don't include that when generating "raw" keys so push has to special case around it. The client JS code (that generates the key in the browser) does use the value. There's some discussion on the Push Server side to preflight various values to help folks get the gnarlier bits right. Push should always accept either the 64 byte raw, or the 65 byte flagged raw, but it's not a bad habit to have data that closely approximates "reality".